### PR TITLE
Fixes #15190: Cloudformation v2

### PR DESCRIPTION
**Pull request by el-k0022**
https://issues.rudder.io/issues/15190

### CODE REVIEW

File: contrib/inventory-hooks/aws-metadata/README.adoc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+= Inventory hooks library
```
Comment (amousset) [inline contrib/inventory-hooks/aws-metadata/README.adoc]: the .adoc and .py files can go directly into the inventory-hooks directory (like the existing one)

File: contrib/inventory-hooks/aws-metadata/README.adoc [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+This folder represents the data awailable on the EC2 instance through a python script. The following url to view all categories of instance metadata frm within a running instance:
+
+----
+http://169.254.169.254/latest/meta-data/
+----
+
+The output of this script is a JSON format.
```
Comment (amousset) [inline contrib/inventory-hooks/aws-metadata/README.adoc]: please give an example

File: contrib/inventory-hooks/aws-metadata/README.adoc (hunk 1 above)
Comment (amousset) [reply contrib/inventory-hooks/aws-metadata/README.adoc]: You copied the README file, but you should base on the cpu_vulnerability.adoc file instead.

**Comment by el-k0022** [comment]
Commit modified

File: contrib/inventory-hooks/aws-metadata/README.adoc [hunk 3]
```diff
@@ -0,0 +1,9 @@
+= Inventory hooks metadata
```
Comment (amousset) [inline contrib/inventory-hooks/aws-metadata/README.adoc]: It should look like https://github.com/Normation/rudder-tools/blob/master/contrib/inventory-hooks/cpu_vulnerabilities.adoc

**Comment by el-k0022** [comment]
Commit modified

**Comment by el-k0022** [comment]
Commit modified

File: contrib/inventory-hooks/aws-metadata/README.adoc (hunk 1 above)
Comment (el-k0022) [reply contrib/inventory-hooks/aws-metadata/README.adoc]: changed

File: contrib/inventory-hooks/aws-metadata/README.adoc (hunk 2 above)
Comment (el-k0022) [reply contrib/inventory-hooks/aws-metadata/README.adoc]: example added.

**Comment by peckpeck** [comment]
What is the difference between this and https://github.com/Normation/rudder-tools/pull/495
Why do you have 2 PR with the same content ?

**Comment by el-k0022** [comment]
There is no difference, I had just removed the first repository by mistake,
and i created this new one.

File: contrib/inventory-hooks/aws-metadata/README.adoc (hunk 3 above)
Comment (amousset) [reply contrib/inventory-hooks/aws-metadata/README.adoc]: The aws-metadata should be removed from the PR now

File: contrib/inventory-hooks/aws.adoc [hunk 4]
```diff
@@ -0,0 +1,70 @@
+= INSTANCE metadata
```
Comment (amousset) [inline contrib/inventory-hooks/aws.adoc]: Title should be more descriptive, something like `AWS EC2 instance metadata`

**Comment by el-k0022** [comment]
Commit modified

File: contrib/inventory-hooks/aws-metadata/README.adoc (hunk 3 above)
Comment (el-k0022) [reply contrib/inventory-hooks/aws-metadata/README.adoc]: removed

File: contrib/inventory-hooks/aws.adoc (hunk 4 above)
Comment (el-k0022) [reply contrib/inventory-hooks/aws.adoc]: recorded

**Comment by el-k0022** [comment]
Commit modified

File: contrib/inventory-hooks/aws.adoc [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+This inventory hook allows gathering information about available data on the EC2 instance, using the following url to wiew all categories of instance metadata from within a running instance: 
+
+----
+http://169.254.169.254/latest/
+----
+
+== Property
+
+This hook provides three properties named `aws-metadata` `aws-dynamic` `aws-user-data` which gets a value of some of the metadata itemswith JSON format.
```
Comment (amousset) [inline contrib/inventory-hooks/aws.adoc]: capitalization is inconsistent

**Comment by el-k0022** [comment]
Commit modified

File: contrib/inventory-hooks/aws.adoc (hunk 5 above)
Comment (el-k0022) [reply contrib/inventory-hooks/aws.adoc]: changed

**Pull request merged**